.github/workflows/scylla.yaml

```diff
@@ -0,0 +1,55 @@
+name: Build Scylla
+
+on:
+  schedule:
+    # 5AM everyday
+    - cron: '0 5 * * *'
+
+permissions: {}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  BUILD_TYPE: RelWithDebInfo
+  BUILD_DIR: build
+
+jobs:
+  build-scylla:
+    runs-on: ubuntu-latest
+    container: fedora:40
+    strategy:
+      matrix:
+        build_type:
+          - Debug
+          - RelWithDebInfo
+          - Dev
+    steps:
+      - run: |
+          sudo dnf -y install git
+      - uses: actions/checkout@v4
+        with:
+          repository: scylladb/scylladb
+          submodules: true
+      - name: Update seastar submodule
+        run: |
+          cd seastar
+          git config --global --add safe.directory '*'
+          git fetch origin master
+          git reset --hard origin/master
+          git submodule update --init --force --recursive
```

Comment:
I think you also need `git submodule sync --recursive` before update in case new submodules were added or removed (it happens very rarely, but still...)

Reply:
fixed.